File: 하루에 한개씩 문제 풀기/Java/Programmers/Level 0/글자 지우기.java
public class Solution {
    public static String solution(String myString, int[] indices) {
        StringBuilder result = new StringBuilder();
        for (int i = 0; i < myString.length(); i++) {
            if (!contains(indices, i)) {
                result.append(myString.charAt(i));
            }
        }
        return result.toString();
    }

    public static boolean contains(int[] arr, int target) {
        for (int num : arr) {
            if (num == target) {
                return true;
            }
        }
        return false;
    }
}

File: 하루에 한개씩 문제 풀기/Go/Programmers/Level 0/글자 지우기.go
func contains(arr []int, target int) bool {
	for _, num := range arr {
		if num == target {
			return true
		}
	}
	return false
}

func solution(myString string, indices []int) string {
	result := ""
	for i, char := range myString {
		if !contains(indices, i) {
			result += string(char)
		}
	}
	return result
}